internal/invites/service.go

```diff
@@ -342,3 +355,9 @@ func getInviteUrl(emailCfg serverconfig.EmailConfig, userInvite db.UserInvite) (
 	}
 	return inviteURL, nil
 }
+
+// isEmail is used to validate if the email is a valid email address according to RFC 5322
+func isEmail(emailString string) bool {
+	_, err := mail.ParseAddress(emailString)
```

Comment:
Should we be using only the `address` part for the rest of the function?  I'm not sure we've thought about `"Joe Bob <target@foo.com>"` and `"Oh gosh, I really dislike you <target@foo.com>"` in the rest of our code.

Reply:
(If so, we could just return `string` or `""`)

Reply:
We can, but do we want to support such kind of inputs? For now at least I assume it's much cleaner to fail if the input is not strictly an email.

Reply:
My point is that `mail.ParseAddress` _will_ accept `"An unpleasant joke from recent politics <target@foo.com>"`, and the `Name` field will be set to `"An unpleasant joke from recent politics"`.

(And I was suggesting we should return just the `parsed.Address` part of the `ParseAddress` result.

Reply:
See https://github.com/bufbuild/protovalidate-go/blob/1446f1e1d576e67382edcb4863f76414e2eb2256/celext/lib.go#L426 for fairly-complete email address validation.

Reply:
(And, I'm okay removing this bit of code in favor of #4840 )

Reply:
I decided to remove it in favour of the validation checks we'll introduce 👍